scripts/model-push/llama-converter/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ignaciolopezluna020/llama.cpp:cpu
+FROM ghcr.io/ggml-org/llama.cpp:full-b5227
```

Comment:
We should also check if we can use `ghcr.io/ggml-org/llama.cpp:light`, we actually only need the tools